doc/developers/contributing.rst

```diff
@@ -112,7 +112,7 @@ following rules before submitting:
    `new algorithm requirements
    <http://scikit-learn.org/stable/faq.html#what-are-the-inclusion-criteria-for-new-algorithms>`_.
 
--  If you are submitting a bug report, we strongly encourage you to follow the guidelines in 
```

Comment:
I know it's already been mentioned to you, but I am going to say it a second time: such unrelated changes are a very bad practice (they create merge conflicts, and destroy the usefulness of "git blame").